ROLES.md

```diff
@@ -276,6 +276,26 @@ The following apply to the area / component for which one would be a lead.
 
   - Mentoring and guiding approvers, members, and new contributors.
 
+## Stepping Down/Emeritus Process
+If and when contributors' commitment levels change, contributors can consider stepping down (moving down the contributor ladder) or moving to emeritus status (completely stepping away from the project).
+
+Once a member has decided to move to emeritus or step down the contributor ladder, their entries in Peribolos should be removed or modified according to the specific change. Members moving to emiritus status should be removed from the Knative GitHub organization in Peribolos through a PR submitted by the member or a member of the Steering Committee.
+
+## Inactivity
+It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project.
+
+* Inactivity is measured by:
+    * Periods of no contributions for longer than 12 months
+    * Periods of no communication for longer than 12 months
+* Consequences of being inactive include:
+    * Involuntary Removal or demotion
+
+## Involuntary Removal or Demotion
+
+Removal or demotion of a contributor happens when responsibilities and requirements aren't being met. This may include repeated patterns of inactivity, extended period of inactivity, a period of failing to meet the requirements of your role, and/or a violation of the Code of Conduct. This process is important because it protects the community and its deliverables while also opens up opportunities for new contributors to step in.
+
+Involuntary removal or demotion is handled through a vote by a majority of the current Steering Committee members.
```

Comment:
Do we want to be a bit more stringent here, maybe requiring 75% in favor of (i.e. a 4-1 vote)? I'd assume removing someone should be fairly unanimous and that if there's two folks opposed we could potentially end up in a tricky spot...

Reply:
Pinging @knative/steering-committee again... 

I can see that a 75% threshold would be good in contentious scenarios, but I imagine most of the time we'll just be purging the ranks of folks who have left the project for one reason or another without voluntarily removing themselves.

Reply:
+1 for a 75% consensus, I also don't see it critical, and most of the time, people probably just "forgot" to remove themselves. Maybe we could add a note that the removed person should be notified about this with a short explanation so that such an involuntary removal does not come with a surprise

Reply:
@rhuss @psschwei I have modified the doc to indicate an 80% vote in favor of removal (4-1) ptal